src/libraries/Microsoft.PowerFx.Interpreter/Functions/Extensions.cs

```diff
@@ -72,7 +72,7 @@ internal static bool CheckAggregateNames(this DType argType, DType dataSourceTyp
                     }
                 }
 
-                if (!dsNameType.Accepts(type, out var schemaDifference, out var schemaDifferenceType, exact: false) &&
+                if (!dsNameType.Accepts(type, out var schemaDifference, out var schemaDifferenceType) &&
```

Comment:
>dsNameType.Accepts(type, out var schemaDifference, out var schemaDifferenceType) [](http://example.com/codeflow?start=21&length=80)

We need to run tests against DV project and check if nothing will broke there. Changes to this file were recently added to fix some problems there.
[https://github.com/microsoft/Power-Fx-Dataverse/issues/102](https://github.com/microsoft/Power-Fx-Dataverse/issues/102)

Reply:
No test failure related to my change on DV

Reply:
As discussed this will be blocked on Issue https://github.com/microsoft/Power-Fx/issues/1205